src/core/operation.c

```diff
@@ -162,6 +185,9 @@ QuicOperationEnqueueFront(
     CXPLAT_DBG_ASSERT(Oper->Link.Flink == NULL);
 #endif
     StartProcessing = CxPlatListIsEmpty(&OperQ->List) && !OperQ->ActivelyProcessing;
+    if (OperQ->PriorityTail == &OperQ->List) {
+        OperQ->PriorityTail = &Oper->Link;
+    }
```

Comment:
@nibanks  What does this for?

Reply:
If you enqueue something at the very front, it is considered a priority item. This is saying that if the priority tail currently is the HEAD, then we need to point the tail after this new item instead.